contents/python/introduction-to-python.md

```diff
@@ -4,25 +4,30 @@ Python is a simple yet powerful and versatile language. Conceived in the late 80
 
 ## Getting started with Python
 
-[Google's Python Class](https://developers.google.com/edu/python/)
+If you're reading this on GitHub, I'm guessing that you're already an experienced programmer looking to get in on the Python action. If that's the case, check out [Google's Python class](https://developers.google.com/edu/python/), which will introduce you to Python's concepts and core data structures like lists, dictionaries, and strings!
```

Comment:
I think you can remove the GitHub part, just because they know GitHub they might not be really experienced programmers....